packages/gatsby-www-components/src/data/sidebars/doc-links.yaml

```
@@ -0,0 +1,702 @@
+            - title: Marketers
+              link: /docs/winning-over-marketers/
+            - title: Executives
+              link: /docs/winning-over-executives/
+            - title: Clients*
+              link: /docs/winning-over-clients/
+            - title: Content Creators
+              link: /docs/winning-over-content-creators/
+        - title: Spreading Gatsby In Different Types of Organizations
+          link: /docs/different-organization-types/
+          breadcrumbTitle: Spreading Gatsby
+          items:
+            - title: As a Freelancer*
+              link: /docs/gatsby-for-freelancers/
+            - title: Inside an Agency
+              link: /docs/gatsby-for-agencies/
+              items:
+                - title: Lower Bids, More Profit
+                  link: /docs/lower-bids-more-profit/
+                - title: Faster Recruiting
+                  link: /docs/faster-recruiting/
+                - title: Sanitize Your Stack
+                  link: /docs/sanitize-your-stack/
+                - title: Why Agencies Go Gatsby-Only
+                  link: /docs/going-gatsby-only/
+            - title: Inside a Company
+              link: /docs/gatsby-for-companies/
+            - title: Inside an Enterprise (Large Company)
+              link: /docs/gatsby-in-the-enterprise/
+              breadcrumbTitle: Inside an Enterprise
+              items:
+                - title: Answering IT/Security Questions
+                  link: /docs/answering-it-security/
+                - title: Setting Up Gatsby Without Gatsby New
+                  link: /docs/setting-up-gatsby-without-gatsby-new/
+        - title: Best Practices for Orgs*
+          link: /docs/best-practices-for-orgs/
+          items:
+            - title: Sharing Components Across Websites
+              link: /docs/sharing-components-across-websites/
+              breadcrumbTitle: Sharing Components
+              items:
+                - title: Component Libraries
+                  link: /docs/component-libraries/
+                - title: Making Components Discoverable*
+                  link: /docs/making-components-discoverable/
+        - title: How Gatsby Boosts Your Career
+          link: /docs/how-gatsby-boosts-career/
+    - title: Contributing
+      link: /contributing/
+      items:
+        - title: Community
+          link: /contributing/community/
+        - title: How to Contribute
+          link: /contributing/how-to-contribute/
+    - title: Partnering With Gatsby
+      link: /docs/partnering-with-gatsby/
+      items:
+        - title: Integrating & Partnering For Vendors
+          link: /docs/gatsby-vendor-partnership/
+        - title: Agency Partnership Program
+          link: /docs/gatsby-agency-partnership/
+    - title: Commands (Gatsby CLI)
+      link: /docs/gatsby-cli/
+    - title: Cheat Sheet
+      link: /docs/cheat-sheet/
+    - title: Glossary
+      link: /docs/glossary/
+      items:
```

fresh content (#19965 #20065) 
```suggestion
      items:
        - title: Node.js
          link: /docs/glossary/node
```

---

fixed